kafka-ui-react-app/src/theme/theme.ts

```diff
@@ -237,6 +252,26 @@ const theme = {
       active: Colors.neutral[50],
     },
   },
+  confirmationModal: {
+    overlay: Colors.transparency[10],
+    shadow: Colors.transparency[20],
+  },
+  icons: {
+    closeIcon: Colors.neutral[30],
+    messageToggleIconOpened: Colors.neutral[90],
+    messageToggleIconClosed: Colors.neutral[30],
+    verticalElipsisIcon: Colors.neutral[50],
+    liveIcon: {
+      circleBig: Colors.red[10],
+      circleSmall: Colors.red[50],
+    },
+  },
+  viewer: {
+    wrapper: Colors.neutral[3],
+  },
+  textarea: {
+    focused: 'transparent',
```

Comment:
As i can see in `kafka-ui-react-app/src/theme/theme.ts` we use `css` pseudo-classes as property names 

So I think `focused` will be better as `focus`